### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by rexmac**
Prep for v2.4.0 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
-## [Unreleased]
+[//]: # '## [Unreleased]'
```
Comment (ismail-syed) [inline CHANGELOG.md]: I thought we generally did the following for all our readmes. At least that's what I've noticed. 

```
<!--## [Unreleased] -->
```

Comment (rexmac) [reply CHANGELOG.md]: 🤷‍♂️ I was just reverting to what it was before.

I definitely prefer the `<!--` syntax to the archaic `[//]: #`.

**Comment by rexmac** [comment]
@ismail-syed I've reverted the version bump and will bump and tag using `yarn version` as you suggested. I've also changed the `Unreleased` comment syntax. :wink:

**Comment by nimzco** [comment]
🎉

**Pull request merged**